app/com/gu/memsub/images/ResponsiveImage.scala

```diff
@@ -1,15 +1,6 @@
 package com.gu.memsub.images
 
 import io.lemonlabs.uri.Uri
-import io.lemonlabs.uri.dsl._
-
-object ResponsiveImageGenerator {
-  def apply(id: String, sizes: Seq[Int], extension: String = "jpg"): Seq[ResponsiveImage] = {
-    sizes.map { size =>
-      ResponsiveImage(id.split("/").fold("https://media.guim.co.uk")({case (a, b) => a / b}) / s"$size.$extension", size)
```

Comment:
this file was failing to compile because of changes in the url library we use, but I realised most of this code is not even used so I've deleted it!